.github/workflows/ci-tests-pytorch.yml

```diff
@@ -22,15 +22,14 @@ on:
       - "!requirements/pytorch/docs.txt"
       - "!*.md"
       - "!**/*.md"
+  schedule:
```

Comment:
What are the implications of this change?

Reply:
In case we don't merge anything for a day, we still get daily glances what is the actual state